package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@cloudquery/plugin-config-ui-lib",
   "description": "Plugin configuration UI library for CloudQuery Cloud App",
-  "version": "0.0.77",
```

Comment:
78?

Reply:
that was the bug I was fixing, accidentally published 78 🫠